fix(kpagination): detect and fix overflow [KHCP-10930] (#2277)
Kong/kongponents: docs/guide/usage.md

  ]
}
```

## Icons

> Kongponents sources all icons from [@kong/icons](https://github.com/Kong/icons) library. If you wish to use standalone icons in your project please refer to the documentation for instructions.

Author: @adamdehaven could you proofread, just in case?

> Kongponents sources all icons from [@kong/icons](https://github.com/Kong/icons) library. If you wish to use standalone icons in your project please refer to the documentation for instructions.

Reviewer: ```suggestion
Kongponents sources its icons from the [@kong/icons](https://github.com/Kong/icons) library. If you wish to use a different icon library in your project, please refer to the library's documentation for instructions.
```

Author: I don't think `@kong/icons` is fully replaceable in Kongponents. What I was trying to say here is that if they wish to use icons exported by `@kong/icons` in their project (as of, outside of Kongponents) they can refer to `@kong/icons` documentation for installation and usage instructions.

Author: Applied
